packages/venia-concept/src/components/Checkout/wrapper.js

```diff
@@ -29,15 +29,15 @@ const isCheckoutReady = checkout => {
     );
 };
 
-class CheckoutWrapper extends Component {
+export class CheckoutWrapper extends Component {
     static propTypes = {
-        beginCheckout: func.isRequired,
-        cancelCheckout: func.isRequired,
+        beginCheckout: func,
```

Comment:
I'm still not clear on whether we want the propType checking to occur at the highest level like here in the wrapper or at the lowest level the prop is utilized.